DOCS-2364 Alerting new documentation structure (#11576)
DataDog/documentation: content/en/monitors/manage/_index.md

---
title: Manage Monitors
kind: documentation
aliases:
    - monitors/manage_monitor/
description: "Send notifications to your teams when monitors trigger alerts"
further_reading:
- link: "/monitors/create/"
  tag: "Documentation"
  text: "Create monitors"
- link: "/monitors/notify/"
  tag: "Documentation"
  text: "Monitor Notifications"
---

Use the [Manage Monitors][1] page to search, delete, mute, or resolve your monitors and edit monitor tags in bulk. You can also clone or edit individual monitors from the search results.

{{< img src="monitors/manage_monitor/monitor_page.jpg" alt="manage monitor page"  >}}

## Search

To [search your monitors][2], construct a query using the facet panel on the left or the search bar at the top.

## Manage

After searching, select one or more monitors to update using the checkboxes next to each result. Select all results with the top checkbox next to the *STATUS* column heading. Modify the monitors in bulk using the buttons at the right above the search results:

| Option    | Description                                                                |
|-----------|----------------------------------------------------------------------------|

> | Mute      | Mute the selected monitors for `1h`, `4h`, `12h`, `1d`, `1w`, or `Forever` |

Reviewer: ```suggestion
| Mute      | Mute the selected monitors for `1h`, `4h`, `12h`, `1d`, `1w`, or `Forever`. |
```

#### Grouped results

The triggered monitors page shows a row for each group (reporting source) of each monitor. For example, a monitor grouped by host with 14 hosts in a triggered status shows 14 rows on the triggered monitors page. This lets you mute or [resolve][3] a monitor for specific reporting sources.

When writing a search query, the same attributes from the manage monitors page are available, even if they are not displayed as check boxes on the triggered monitors page.

Attribute differences for the triggered monitors page:

* `group_status` is the attribute name instead of `status`.
* The `triggered` attribute lets you filter monitors by how long they have been triggered.
* The `group` attribute helps you narrow down search results for monitors grouped by more than one tag. For example, a monitor is grouped by `host` and `env`. After searching, you see four rows with the groups `host:web01,env:dev`, `host:web02,env:dev`, `host:web01,env:prod`, and `host:web02,env:prod`. Use the `group` attribute to only show prod hosts (`group:"env:prod"`) or web02 hosts (`group:"host:web02"`).

### Monitor tags

> Monitor tags are independent of tags sent by the Agent or integrations. Add up to 32 tags directly to your monitors for filtering on the [manage monitors][1], [triggered monitors][5], or [manage downtime][6] pages. Learn more about monitor tags in the [documentation about tags][7].

Reviewer: ```suggestion
Monitor tags are independent of tags sent by the Agent or integrations. Add up to 32 tags directly to your monitors for filtering on the [manage monitors][1], [triggered monitors][5], or [manage downtime][6] pages. Learn more about monitor tags in [Assigning Tags for the UI][7].
```

> **Note**: Those tags are added to the alert event generated by the monitor, up to 25 tags in total on the alert event.

Reviewer: ```suggestion
**Note**: Monitor tags are added to the alert event generated by the monitor, up to 25 tags in total on the alert event.
```
